msibi/tutorials/README

```diff
@@ -5,4 +5,9 @@ code works as expected.
 LJ : recover known LJ potential
     - This test, as described in the original paper, is recovering a
       known potential to show that the methodology works. The target
-      data was derived from LJ fluids at three different states. 
+      data was derived from LJ fluids at three different states.
+
+PROPANE : recover known propane potential
+    - This test, as described in the original 2014 paper, is recovering a
+      known potential to show that the methodology works. The target
```

Comment:
👍 . And the heading should be something like

PROPANE : optimize pair potentials for a single-site propane model